CIP-0001/README.md

```diff
@@ -195,105 +133,40 @@ CIP editors serve as stewards of the Cardano ecosystem, here to support and prog
+### Progression
 
-Some CIPs receive exposure outside the development community prior to completion, and other CIPs might not be completed at all. To avoid a situation where critical CIP reviews may go unnoticed during this period, reviewers may, at their option, still post their review on the comments page, provided they first post it to the forum and plan to later remove or revise it as applicable based on the completed version. Such revisions should be made by editing the previous review and updating the timestamp. Reviews made prior to the complete version may be removed if they are no longer applicable and have not been updated in a timely manner.
+Proposals must be submitted to the [cardano-foundation/CIPs](https://github.com/cardano-foundation/CIPs/pulls) repository as a pull request named after the proposal's title. Ideally, the community should have already discussed a proposal before its submission. Early reviews streamline the process down the line -- although it isn't a strict requirement.
 
-Pages must be named after the full CIP number (eg, "CIP 0001") and placed in the "Comments" namespace.
-For example, the link for CIP 1 will be <https://github.com/cardano-foundation/CIPs/wiki/Comments:CIP-0001>.
+CIP editors meet regularly (on a 2-week cadence) in [a public Discord server](https://discord.gg/qd6jE9Xj) to go through newly proposed ideas in a "triage" phase. As a result of a triage, editors acknowledge new CIPs, and briefly review their preamble. It is recommended not yet to assign a number and pick `?` when first opening a proposal. Instead, editors will allocate a temporary number as part of the triage phase. The triage also allows new CIPs to get visibility for potential reviews.
 
-Comments posted to this wiki should use the following format:
+In every meeting, editors will also review in more depth some chosen CIPs (based on their readiness and the stability of the discussions about them). The goal of a review is to assess points mentioned earlier and, in particular, judge whether a CIP is reaching a consensus within the community. A proposal must also have an approved (by the editors) path to 'Active', demonstrating the adoption of the CIP to move forward. Each proposal is unique and has a bespoke path to 'Active', which must then be reviewed case-by-case.
 
-    <Your opinion> --<Your name>, <Date of posting, as YYYY-MM-DD>
+Once accepted, CIPs are merged with the status `Proposed` until they meet their path to 'Active' requirements. In some cases (mainly in the case of informational CIPs), proposals may be merged as `Active` immediately. In general, however, there must be sufficient time between the first appearance of a proposal and its acceptance into the repository to ensure enough opportunity for community members to review it.
```

Comment:
Just checking, I think this is a change in the process, suggesting that every merged CIP should be at least in Proposed status, and thing shouldn't be merged as Draft?

Reply:
So presumably a CIP PR should set the status to `Proposed` and the fact of it being a PR means it's a draft?

Reply:
What you're describing is the process as outlined in CIP-0001 **currently**; but it does not reflect what is actually happening. In practice, we tend to not merge CIP as draft (anymore), but keep on iterating until they're good enough to be merged as _proposed_. There are some exceptions of course, but generally speaking, the drafting stage happens in the PR. 

This is something we want to change in the long-run of course (and use for example github discussions instead, which have threads) and I don't particularly like it. But as the title's of this PR indicates, this is meant to update CIP-0001 to reflect reality.

Reply:
> This is something we want to change in the long-run of course

Why? What's so bad about the drafting process corresponding to the PR stage? That's a process that people are very familiar with, whereas the idea that something gets merged while still a draft is more unusual. I certainly found it confusing.

Reply:
Discussions on pull requests are hard to manage and keep track of. Especially on long running CIPs with hundreds of comments. There tends to be cross chats, and as the underlying proposal gets updated, some comments become obsolete making it even harder to follow.

Although, that's also 'the best we have' for now. So maybe all that is needed is some more discipline / practices in the PR / Comment management. I don't know. Nothing set in stone yet, but I can see that there's something that _could_ be improved.